### PR TITLE
Added DscJobs

### PR DESCRIPTION
**Pull request by AhmedSabry957**
image
image
image
image

### CODE REVIEW

**Comment by AhmedSabry957** [comment]
I added a new pull request with new codes as the owner of DscJobs, some problems happened with the other contributor,
I improved the codes, check and tell me if there anything needs to be changed

File: websites/D/Dscjobs.org/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Dexter",
+    "id": "780079091172900884"
+  },
+  "service": "Dscjobs.org",
```
Comment (OGMatrix) [inline websites/D/Dscjobs.org/dist/metadata.json:6]: Official Name is **DscJobs**

**Review by OGMatrix — CHANGES_REQUESTED** [review]

File: websites/D/DscJobs/presence.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+  clientId: "807748912940711996"
+});
+
+var time = Math.floor(Date.now() / 1000);
+var path = window.location.pathname;
```
Comment (Bas950) [inline websites/D/DscJobs/presence.ts:6]: no var

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by AhmedSabry957** [comment]
Alright fixed

**Review by Bas950 — CHANGES_REQUESTED** [review]
Squash commits

File: websites/D/DscJobs/presence.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  };
+
+  switch (path) {
+    case '/': presenceData.details = "Viewing Home Page"; break;
+    case '/premium': presenceData.details = "Viewing Premium Page"; break;
+    case '/create': presenceData.details = "Creating Profile"; break;
+    case '/legal': presenceData.details = "Viewing Legal Page"; break;
+    case '/partners': presenceData.details = "Viewing Partners Page"; break;
+    case '/supporters': presenceData.details = "Viewing Supporters Page"; break;
+    case '/moderators': presenceData.details = `Viewing Moderators Page ${location.href.split("page=")[1].split("&")[0]}`; break;
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:20]: I don't think it will look good when it will be like: `Viewing Moderators Page 1`

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/D/DscJobs/presence.ts (hunk 3 above)
Comment (AhmedSabry957) [reply websites/D/DscJobs/presence.ts]: I changed it

Comment (AhmedSabry957) [reply websites/D/DscJobs/presence.ts]: only this and it will be approved?

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**